src/main/java/com/checkmarx/flow/service/EmailService.java

```diff
@@ -39,34 +46,36 @@ public EmailService(FlowProperties properties, TemplateEngine templateEngine, Ja
      * @param ctx
      * @param template
      */
-    public void sendmail(List<String> recipients, @NotNull String subject, @NotNull Map<String, Object> ctx, String template){
-        if(ScanUtils.empty(recipients)){
-            log.info("Email not sent - no recipients listed");
-            return;
-        }
+    public void sendmail(List<String> recipients, @NotNull String subject, @NotNull Map<String, Object> ctx, String template) {
+
         MimeMessagePreparator messagePreparator = mimeMessage -> {
-            log.info("Sending email");
+            log.info("Sending email notification.");
             MimeMessageHelper messageHelper = new MimeMessageHelper(mimeMessage);
 
-            String contact = properties.getContact();
-            if(!ScanUtils.empty(contact)){
+            String contact = flowProperties.getContact();
+            if (!ScanUtils.empty(contact)) {
```

Comment:
It's better to use org.apache.commons.lang3.StringUtils#isEmpty

Reply:
Done.